law.m.aaron/wwwroot/js/game/GameScene.js

```diff
@@ -211,7 +216,9 @@
      * */
     delayDone() {
```

Comment:
You can remove this function delayDone and the code that calls it since it currently does nothing.

Reply:
Removed it.